ether-wake.c

```diff
@@ -23,6 +23,7 @@ static char usage_msg[] =
 "		-b	Send wake-up packet to the broadcast address.\n"
 "		-D	Increase the debug level.\n"
 "		-i ifname	Use interface IFNAME instead of the default 'eth0'.\n"
+"		-d ipaddress	Defer delivery of matched packages until host with IPADDRESS responds to a ping i.e. has woken up.\n"
```

Comment:
Could you replace the word 'packages' by 'packets' (referring to TCP or UDP packets here). Also, could you add a line break somewhere to make this line approx. as long as the other lines.